src/components/emptyState/index.js

```diff
@@ -1,4 +1,5 @@
 import NoResults from './NoResults';
+import NoSatelliteManifests from './NoSatelliteManifests';
 import Processing from './Processing';
 
-export { NoResults, Processing };
+export { NoResults, NoSatelliteManifests, Processing };
```

Comment:
We may want to think about relocating and/or renaming some of these empty states because the distinction between `NoResults` and `NoSatelliteManifests` is not immediately apparent.

Reply:
Yeah I actually had renamed NoResults to NoSearchResults but then I reverted back to cut down on changes/semi-related refactors.  

I think NoSearchResults (old NoResults) and CreateManifestPanel (old NoSatelliteManifests) are good names; I just pushed that change; lmk thoughts.

Reply:
I think they could use more rework in the future, but this accomplishes differentiating them better now.